frontend/src/components/translation/TranslationTable.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import "./TranslationTable.css";
+import { Badge, Box, Button, Flex } from "@chakra-ui/react";
+import Cell from "./Cell";
+import EditableCell from "./EditableCell";
+import { StoryLine } from "./Autosave";
+
+export type TranslationTableProps = {
+  translatedStoryLines: StoryLine[];
+  disabled?: boolean;
```

Comment:
Thoughts on naming this `editable` instead of `disabled`? I think captures the idea of "if translation is editable, use editable cells else use cells" a bit better